README.md

```diff
@@ -1,21 +1,29 @@
 # rebar3_ex_doc
 
-[![Build Status](https://github.com/starbelly/rebar3_ex_doc/actions/workflows/ci.yml/badge.svg)](https://github.com/starbelly/rebar3_ex_doc/actions/workflows/ci.yml) [![Erlang Versions](https://img.shields.io/badge/Supported%20Erlang%2FOTP-%2024-blue)](http://www.erlang.org)
+[![Build Status](https://github.com/starbelly/rebar3_ex_doc/actions/workflows/ci.yml/badge.svg)](https://github.com/starbelly/rebar3_ex_doc/actions/workflows/ci.yml)
+[![Erlang/OTP Version](https://img.shields.io/badge/erlang%2Fotp-%2024-blue)](http://www.erlang.org)
+[![Hex Version](https://img.shields.io/hexpm/v/rebar3_ex_doc.svg)](https://hex.pm/packages/rebar3_ex_doc)
+[![Hex Docs](https://img.shields.io/badge/hex-docs-lightgreen.svg)](https://hexdocs.pm/rebar3_ex_doc/)
+[![Total Download](https://img.shields.io/hexpm/dt/rebar3_ex_doc.svg)](https://hex.pm/packages/rebar3_ex_doc)
+[![License](https://img.shields.io/hexpm/l/rebar3_ex_doc.svg)](https://github.com/starbelly/rebar3_ex_doc/blob/master/LICENSE)
+[![Last Updated](https://img.shields.io/github/last-commit/starbelly/rebar3_ex_doc.svg)](https://github.com/starbelly/rebar3_ex_doc/commits/master)
 
-rebar3_ex_doc is a rebar3 plugin to generate documentation using [ex_doc](https://github.com/elixir-lang/ex_doc). This
-plugin ships with ex_doc as an escript, thus you don't need to worry about having Elixir installed nor compiling
-the ex_doc dependencies.
+`rebar3_ex_doc` is a `rebar3` plugin to generate documentation using [ex\_doc](https://github.com/elixir-lang/ex_doc). This plugin ships with `ex_doc` as an escript, thus you don't need to worry about having Elixir installed nor compiling the `ex_doc` dependencies.
+
+<div style="text-align: center;">
```

Comment:
Do you think the screenshot would be better if we trimmed the browser parts out (e.g,., the url bar, etc.) ?

Reply:
I'll update the screenshot to remove the border. FYI, I added the browser parts in the first place because the captured screenshot was lacking border and looked a bit out of place. See screenshot below:

![20220119_060918](https://user-images.githubusercontent.com/134518/150026804-8cd22382-cdd8-4b2b-8033-37337867e5ff.png)